dotCMS/src/integration-test/java/com/dotcms/publisher/util/DependencyModDateUtilTest.java

```diff
@@ -0,0 +1,374 @@
+     * @throws DotDataException
+     */
+    @Test
+    @UseDataProvider("dataProviderForcePush")
+    public void excludeByModDateContentWithPushAssetAfterModDate(final TestCase testCase) throws DotDataException {
+        final Environment environment_1 = new EnvironmentDataGen().nextPersisted();
+        final Environment environment_2 = new EnvironmentDataGen().nextPersisted();
+        final PushPublishingEndPoint endPoint = new PushPublishingEndPointDataGen()
+                .environment(environment_1)
+                .nextPersisted();
+
+        final Bundle testBundle = createTestBundle(testCase.isForcePush, Collections.singletonList(environment_1));
+
+        PushPublisherConfig config = createPushPublisherConfigMock(
+                testBundle, testCase.isDownloading, testCase.isStatic, testCase.operation);
+
+        final DependencyModDateUtil dependencyModDateUtil = new DependencyModDateUtil(config);
+
+        final Host host = new SiteDataGen().nextPersisted();
+        final Language language = new LanguageDataGen().nextPersisted();
+
+        final ContentType contentType =  new ContentTypeDataGen()
+                .host(host)
+                .nextPersisted();
+
+        final Calendar yesterday = Calendar.getInstance();
+        yesterday.add(Calendar.DATE, -1);
+
+        final Contentlet contentlet =  new ContentletDataGen(contentType.id())
+                .languageId(language.getId())
+                .host(host)
+                .modeDate(yesterday.getTime())
+                .nextPersisted();
+
+        final Bundle bundle = new BundleDataGen()
+                .pushPublisherConfig(config)
+                .downloading(testCase.isDownloading)
+                .addAssets(Arrays.asList(contentlet))
+                .operation(testCase.operation)
+                .forcePush(testCase.isForcePush)
+                .nextPersisted();
+
+        new PushedAssetDataGen()
+                .assetId(contentlet.getIdentifier())
+                .assetType(PusheableAsset.CONTENTLET.toString())
+                .bundle(bundle)
+                .publishingEndPoint(endPoint)
+                .environment(environment_1)
+                .pushDate(new Date())
+                .publisher(PushPublisher.class)
+                .nextPersisted();
+
+        final boolean excludeByModDate = dependencyModDateUtil.excludeByModDate(contentlet, PusheableAsset.CONTENTLET);
+
+        if ( testCase.isForcePush || testCase.isDownloading ||
+                testCase.operation != Operation.PUBLISH ) {
+
+            assertFalse(excludeByModDate);
+        } else {
+            assertTrue(excludeByModDate);
+        }
+    }
+
+    /**
+     * Method to Test: {@link DependencyModDateUtil#excludeByModDate(Object)}}
+     * When: A contentlet with two environment:
+     * - One environment with a  PushAsset with date after that moddate
+     * - One environment with a  PushAsset with date before that moddate
+     *
+     * Should: always return true
```

Comment:
return false, or do not exclude

Reply:
done https://github.com/dotCMS/core/pull/20521/commits/078b38ddfdf91f36ed147df16700c696a0da57c1#diff-b2d9b2ef3027cf0691e8532ead5ebd45917b52beae669c039b06fb14eef45600R258